ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -58,25 +58,26 @@
                   stops))))
 
 (extend-protocol tuck/Event
-  LoadStops
+  InitRoute
```

Comment:
I know that this seems to be the way to go right now, but it is actually good thing that these functions are separated.
When saved route is loaded, those route points needs to be loaded without reseting loaded data (from db).

Reply:
The determination of what needs to done differently in editing vs new scenarios should be done in the controller. So the InitRoute is a good name for that, but could be split by making some functions it calls.